wagtailautocomplete/tests/test_edit_handlers.py

```diff
@@ -21,10 +25,13 @@ def setUp(self):
         model = House  # a model with a foreign key to Person
 
         # a AutocompletePanel class that works on House's 'owner' field
-        self.edit_handler = (
-            ObjectList([AutocompletePanel('owner')])
-            .bind_to(model=House, request=self.request)
-        )
+        self.edit_handler = ObjectList([AutocompletePanel("owner")])
+        if WAGTAIL_VERSION >= (3, 0):
+            self.edit_handler = self.edit_handler.bind_to_model(House)
+        else:
+            self.edit_handler = self.edit_handler.bind_to(
+                model=House, request=self.request
+            )
```

Comment:
Note: I see what you're trying to do here, I think it's more confusing to assign the object list and then reassign the bound model to the same attribute instead of just doing a straight assignment with different methods. What do you think?

Reply:
I have altered all the similar conditions in this file.

Much nicer 👍